jgrapht-core/src/main/java/org/jgrapht/generate/GnmRandomBipartiteGraphGenerator.java

```diff
@@ -194,6 +197,22 @@ public void generateGraph(
 
     }
 
+    /**
+     * Returns the first partition of vertices in the bipartite graph.
+     * @return one partition of the bipartite graph
+     */
+    public Collection<V> getFirstPartition(){
```

Comment:
I've made it a LinkedHashSet to ensure that the iteration order always remains the same